### DISCUSSION TITLE
Searching an entries listing in the CP resets the results

### DISCUSSION THREAD

**Discussion by j-greig** [craftcms/cms#8533] [Ideas]
### Description
See http://recordit.co/8v9OVaDJda for a screencast of the issue.

### Steps to reproduce

1. Scroll down the entries list at /admin/entries/blah to load additional entries
2. Perform a search in the entries search field
3. Entries list is now reset to the default number of entries

### Additional info

- Craft version:  2.6.3015
- PHP version: 7.1
- Database driver & version: MySQL (whatever the default version on Forge/Linode is)

**Comment by j-greig** [comment]
PS. Maybe this is meant to happen, but to me it feels like an unexpected behaviour.

I guess https://github.com/craftcms/cms/issues/818 would solve this :)

**Comment by brandonkelly** [comment]
Yes, this is (currently) expected behavior – when you search you are basically running a new element query with a new set of parameters (now including `search`), and each time the query changes, you’re getting a new set of results.

#818 wouldn’t help. But maybe Craft could start preserving the pre-search view state, and restore it when you clear the search query.

**Comment by j-greig** [comment]
Understood, thank you for clarifying!
